MAINTAINERS

```diff
@@ -1,597 +1,33 @@
-# More detail on Moby project governance is available in the
-# project/GOVERNANCE.md file found in this repository.
+# See project/GOVERNANCE.md for committer versus reviewer roles
 #
-# It is structured to be consumable by both humans and programs.
-# To extract its contents programmatically, use any TOML-compliant
-# parser.
+# COMMITTERS
+# GitHub ID, Name, Email address, GPG fingerprint
+"akerouanton","Albin Kerouanton","albinker@gmail.com"
+"AkihiroSuda","Akihiro Suda","akihiro.suda.cz@hco.ntt.co.jp"
+"austinvazquez","Austin Vazquez","macedonv@amazon.com"
+"cpuguy83","Brian Goff","cpuguy83@gmail.com"
+"robmry","Rob Murray","rob.murray@docker.com"
+"thaJeztah","Sebastiaan van Stijn","github@gone.nl"
+"tianon","Tianon Gravi","admwiggin@gmail.com"
+"tonistiigi","Tõnis Tiigi","tonis@docker.com"
+"vvoland","Paweł Gronowski","pawel.gronowski@docker.com"
 #
-# TODO(estesp): This file should not necessarily depend on docker/opensource
-# This file is compiled into the MAINTAINERS file in docker/opensource.
-#
-[Org]
-
-	[Org."Core maintainers"]
-
-	# The Core maintainers are the ghostbusters of the project: when there's a problem others
-	# can't solve, they show up and fix it with bizarre devices and weaponry.
-	# They have final say on technical implementation and coding style.
-	# They are ultimately responsible for quality in all its forms: usability polish,
-	# bugfixes, performance, stability, etc. When ownership  can cleanly be passed to
-	# a subsystem, they are responsible for doing so and holding the
-	# subsystem maintainers accountable. If ownership is unclear, they are the de facto owners.
-
-		people = [
-			"akerouanton",
-			"akihirosuda",
-			"anusha",
-			"austinvazquez",
-			"coolljt0725",
-			"corhere",
-			"cpuguy83",
-			"crazy-max",
-			"estesp",
-			"johnstep",
-			"justincormack",
-			"kolyshkin",
-			"laurazard",
-			"mhbauer",
-			"neersighted",
-			"robmry",
-			"rumpl",
-			"runcom",
-			"samuelkarp",
-			"stevvooe",
-			"thajeztah",
-			"tianon",
-			"tibor",
-			"tonistiigi",
-			"unclejack",
-			"vdemeester",
-			"vieux",
-			"vvoland",
-			"yongtang"
-		]
-
-	[Org.Curators]
-
-	# The curators help ensure that incoming issues and pull requests are properly triaged and
-	# that our various contribution and reviewing processes are respected. With their knowledge of
-	# the repository activity, they can also guide contributors to relevant material or
-	# discussions.
-	#
-	# They are neither code nor docs reviewers, so they are never expected to merge. They can
-	# however:
-	# - close an issue or pull request when it's an exact duplicate
-	# - close an issue or pull request when it's inappropriate or off-topic
-
-		people = [
-			"alexellis",
-			"andrewhsu",
-			"bsousaa",
-			"dmcgowan",
-			"fntlnz",
-			"gianarb",
-			"olljanat",
-			"programmerq",
-			"ripcurld",
-			"sam-thibault",
-			"samwhited",
-			"thajeztah",
-			"thompson-shaun",
-		]
-
-	[Org.Alumni]
```

Comment:
This can be kept?

Reply:
Yeah, it will be added in another PR. Some changes to project files will be done after governance. This section needs a time consuming re-format and addition for more people, but its not related to governance now. I'm not proposing an alumni role like containerd has, that doesn't exist here now but could be considered later.